hackernews.el

```diff
@@ -75,7 +75,8 @@
   'hackernews-items-per-page "0.4.0")
 
 (defcustom hackernews-items-per-page 20
-  "Default number of stories to retrieve in one go."
+  "Default number of stories to retrieve in one go.
+If nil, the stories will fill the window."
```

Comment:
For future reference, changes to user options should be accompanied by updates to their `:type` and `:version`/`:package-version` tags.